Weapons.cat

```diff
@@ -447,29 +447,7 @@
         </infoLink>
       </infoLinks>
     </selectionEntry>
-    <selectionEntry type="upgrade" import="true" name="Power sword" hidden="false" id="59a9-57c1-fff9-ca75">
```

Comment:
Something has gone wrong with this merge because power sword's profile was removed